### DISCUSSION TITLE
Co-app is not found

### DISCUSSION THREAD

**Discussion by singularity70** [aclap-dev/video-downloadhelper#1351] [Help and Q&A]
### Your question

Is there a solution for the not-found problem on High-Sierra?

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-GB
coapp: {"found":false,"error":"An unexpected error occurred"}
license: {"unset":true}
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:109.0) Gecko/20100101 Firefox/115.0
{
    "used_history_button": true,
    "download_directory": "/Users/home/dwhelper",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 1,
    "successfull_dl": 97
}

**Comment by Mattmjs** [comment]
What version is your Mac OS ?

**Reply by singularity70** [reply]
(10.13.5)
found the old version that works. 
https://addons.mozilla.org/firefox/downloads/file/4157053/video_downloadhelper-8.0.0.6.xpi
solved.
